go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/google/go-cmp v0.5.6
+	github.com/nightlyone/lockfile v1.0.0
```

Comment:
another advantage of using the build cache is that we don't need to do any locking, as its system takes care of that for us.